### PR TITLE
Fix context names in README

### PR DESCRIPTION
**Pull request by raphael-proust**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ The contexts are configured in the `dune-workspace` file.
 For instance, the following commands will execute `test.exe` in the build context `profile` and `profile-auto`.
```
Comment (nojb) [inline README.md]:
```suggestion
For instance, the following commands will execute `test.exe` in the build context `profiling` and `profiling-auto`.
```

**Review by nojb — COMMENTED** [review]
Thanks! I think there are a few other places where the context name needs to be fixed. Could you check if there are any other places apart from the ones I noticed below?

**Comment by raphael-proust** [comment]
Good catch!  
I went through the files and I didn't notice any other places that need the same change.

**Review by nojb — APPROVED** [review]
LGTM

**Comment by nojb** [comment]
Thanks!

**Pull request merged**